### PR TITLE
Basic UI updates

### PR DESCRIPTION
**Pull request by VargBurz**
This PR fixes displaced titles in inline mode && updates basic UI.

## Changes:
  - update UI using the 8-pt grid
  - add line background
  - rename classes
  - refactor styles
  - other changes will be added soon

### Before:

image

image

### After:

image

image

### CODE REVIEW

**Comment by jonyrock** [comment]
Thanks @mattchaley for help here

File: src/css/panel.base.css [hunk 1]
```diff
@@ -5,37 +5,48 @@
 
 .progress-bar {
   position: relative;
-  height: 34px;
+  height: 32px;
```
Comment (jonyrock) [inline src/css/panel.base.css:8]: Would be nice to make it configurable later

File: src/css/panel.base.css [hunk 2]
```diff
@@ -5,37 +5,48 @@
...
 
 .progress-bar .progress-bar-title {
-  margin-bottom: -8px;
+  margin-bottom: 0px;
+  margin-left: 4px;
 }
 
 .options-tab .section {
-  margin-right: 1rem
+  margin-right: 1rem;
```
Comment (jonyrock) [inline src/css/panel.base.css:51]: I afraid of using `px` together with `rem`, but I hope you guys know what you are doing here

File: src/progress_bar.ts [hunk 3]
```diff
@@ -6,8 +6,8 @@ import * as _ from 'lodash';
 
 type ProgressTitle = {
   barHeight: number,
-  titleTopMargin: number,
-  valueTopMargin: number
+  titleHeight: number,
+  position: string
```
Comment (jonyrock) [inline src/progress_bar.ts:10]: please make an type (enum maybe) here

**Review by jonyrock — CHANGES_REQUESTED** [review]

**Review by rozetko — APPROVED** [review]

**Pull request merged**